api/app/users/api.py

```diff
@@ -391,20 +322,21 @@ class PasswordResetRequestAPI(restful.Resource):
 
     def post(self):
 
-        req_parser=reqparse.RequestParser()
+        req_parser = reqparse.RequestParser()
         req_parser.add_argument('email', type=str, required=True)
-        args=req_parser.parse_args()
+        args = req_parser.parse_args()
 
         LOGGER.debug(
             "Requesting password reset for email {} and organisation {}".format(args['email'], g.organisation.name))
 
-        user = user_repository.get_by_email(args['email'], g.organisation.id)
+        user = db.session.query(AppUser).filter(
```

Comment:
Please change this back to use the user_repository function, because we need to filter users by organisation as well

Reply:
Ah okay, my bad, will fix it.

Reply:
Done.